app/controllers/people_controller.rb

```diff
@@ -15,7 +15,8 @@ def index
   # GET /people/1
   # GET /people/1.json
   def show
-    @company = Company.new
+    @interactions = Interaction.all.order(created_at: :desc).page(params[:page]).where(user_id: current_user.id).where(person_id: params["id"])
```

Comment:
Can we use the @person object and call `@person.interactions` vs the all / where clause